models/cart/src/shipping-info/types.ts

```diff
@@ -0,0 +1,11 @@
+import type { ShippingInfo } from '@commercetools/platform-sdk';
+import type { TBuilder } from '@commercetools-test-data/core';
+
+export type TShippingInfo = ShippingInfo;
+
+export type TShippingInfoGraphql = TShippingInfo & {
```

Comment:
The Graphql representation has different value types for several fields and also has some extra ones so we need to define them here.

![image](https://github.com/user-attachments/assets/50e52af9-40b8-4c37-968e-6dbfc564099d)